packages/cardano-services/test/entrypoints.test.ts

```diff
@@ -114,17 +162,15 @@ describe('entrypoints', () => {
           await ogmiosServerReady(ogmiosConnection);
         });
 
-        it('cli:start-server exposes a HTTP server at the configured URL with all services attached', async () => {
+        it('exposes a HTTP server at the configured URL with all services attached when using CLI options', async () => {
```

Comment:
missing case for `enable-metrics`

Reply:
Done in 5172447ed17514d9dfb6b5e398b8f6a7e03044b5 @iadmytro